images/bin/Makefile

```diff
@@ -1,4 +1,4 @@
-tag = 1.2.18
+tag = 1.2.19
```

Comment:
I'll build the image  post merge.

Reply:
You are missing the change here:
https://github.com/okteto/okteto/blob/705b52ee2be1ae410cba21df52b7c720d953c03a/pkg/model/dev.go#L97